data_steward/utils/participant_summary_requests.py

```diff
@@ -104,4 +123,35 @@ def get_deactivated_participants(project_id, columns):
     df = pandas.DataFrame(deactivated_participants,
                           columns=deactivated_participants_cols)
 
-    return df
+    # To store dataframe in a BQ dataset table
+    destination_table = dataset_id + '.' + tablename
+
+    dataset = store_participant_data(df, project_id, destination_table)
+
+    return dataset
+
+
+def store_participant_data(df, project_id, destination_table):
+    """
+    Stores the fetched participant data in a BigQuery dataset. If the
+    table doesn't exist, it will create that table. If the table does
+    exist, it will append the data onto that designated table.
+
+    :param df: pandas dataframe created to hold participant data fetched from ParticipantSummary API
+    :param project_id: identifies the project
+    :param destination_table: name of the table to be written in the form of dataset.tablename
+
+    :return: returns a dataset with the participant data
+    """
+
+    # Parameter check
+    if not project_id or not isinstance(project_id, str):
+        raise RuntimeError(
+            f'Please specify the project in which to create the tables')
+
+    stored_dataset = pandas_gbq.to_gbq(df,
```

Comment:
Is pylint is actually hitting the API and returning the deactivated participants dataframe generated in `get_deactivated_participants`?

Reply:
No.  Pylint is a static analysis tool, (meaning it doesn't run your software).  This is the `pandas_gbq` [documentation](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.to_gbq.html).  If you follow the source link, you can see it doesn't return a value.  Pylint marked it as an error because you are trying to assign a value from a function that doesn't return a value.

Reply:
Gotcha, I removed the value assignment so `store_participant_data` is returning `pandas_gbq.to_gbq(...)`. Ran pylint and no longer getting that error.